FrontMen/fm-hours
# Feat: timesheets collection

# Changes

## Related issues

Resolver #23 

## Added

- Timesheets collection to hold the status data
- New timesheets page table showing the status of several weeks and navigate directly to that referred week on click

## Removed

- Status field on records

## Changed

- Logic to save and fetch records keeping reference on timesheets

## How to test

- Check the full cycle of the application, create records, save them, approve, deny, change, delete, etc

## Screenshots

![Screen Shot 2021-05-15 at 04 22 59](https://user-images.githubusercontent.com/45885054/118345762-eb91e280-b536-11eb-9343-faef7abf4f37.png)
